content/2-4.md

```diff
@@ -61,54 +61,59 @@ Skills required for this module.
-* mark up tables using the HTML element `table`
-* mark up and identify table headers using the HTML element `th`
-* mark up and identify table data cells using the HTML element `td`
-* explain how table markup allows text-to-speech technologies to associate information in header and data cells
-* explain how table markup allows tables to be displayed using custom style sheets or different screen sizes
+* explain how table markup allows assistive technologies, such as text-to-speech, to associate information in header and data cells
+* explain how table markup supports adaptive strategies, such as displaying tables using custom style sheets and on different screen sizes
+* identify tables that present data and mark up such data tables using the HTML element `table`
+* identify and mark up table headers using the HTML element `th`
+* identify and mark up table data cells using the HTML element `td`
+* identify tables that are used for layout purposes rather than to present data, and code such layout using CSS techniques instead of using HTML structures
 
 #### Teaching Ideas for Topic
 
 Optional ideas to teach the learning outcomes:
 
 * Define simple tables as those which contain one row or column header. Explain that speech and mainstream technologies can associate information in the header and data cells if they are marked up appropriately. Examples of simple tables are provided in the WAI tutorials on [Tables with one header](https://www.w3.org/WAI/tutorials/tables/one-header/).
 * Show examples of use of the HTML elements `table`, `th`, and `td`. Explain that `table` is used to mark up the table layer, `th` is used to code header cells, and `td` is used to code data cells. For examples of how to mark up a simple table, refer to technique [H51: Using table markup to present tabular data](https://www.w3.org/WAI/WCAG21/Techniques/html/H51).
-* Demonstrate use of voice commands, keystrokes, or gestures provided by speech and mainstream technologies to navigate web pages through tables, and to move between cells in a table. Explore advanced functionality that some tools provide, such as presenting all tables in a list where users can select the table they are interested in. Examples of how people with disabilities interact with the web are provided in [Stories of Web Users](https://www.w3.org/WAI/people-use-web/user-stories/).
-* Provide examples of alternative renderings for tables, such as splitting the table into several lists, or showing only parts of the table in a mobile viewport.
+* Demonstrate the use of voice commands, keystrokes, or gestures provided by speech and mainstream technologies to navigate tables, including moving between header and data cells. Explore advanced functionality that some tools provide, such as presenting tables as linear information. Examples of how people with disabilities interact with the web are provided in [Stories of Web Users](https://www.w3.org/WAI/people-use-web/user-stories/).
```

Comment:
I think this might benefit from a specific link to a story that relates to tables. I think the only reference to tables is for Ilya and that is only as part of a longer list of things.

Reply:
I am not sure what the added value of such a link would be, especially given that tables are mentioned there at the same level of other structural components.

I ended up resorting to the overall link to how people with disabilities use the web (same as we do in other modules) because the stories are structured according to the type of disabilities rather than according to the type of components discussed, and that is totally understandable given the context and audience of that resource.

Open to re-explore this, but we may have a hard time trying to find a 1-1 relationship.

Reply:
I agree that 1-1 relationship would be difficult. The trouble is that where there is a reference to the Stories of Web Users resource from a specific point, the reader may be looking for something that expands upon that specific point. If no such content exists then I think it may be better to drop the specific references and keep it general.